.github/workflows/container-scan.yml

```diff
@@ -0,0 +1,37 @@
+name: "Scan containers"
+on:
+  schedule:
+    # every night at one
+    - cron:  '0 1 * * *'
```

Comment:
This is too frequent - the bleeding edge is only updated once a week. The releases are updated once a month

Reply:
The CVE database is updated more frequently. To find new found issues in existing images, we need to run it more frequent than the releases are created.

Reply:
NTC